components/ActionSheet/src/MDCActionSheetController.h

```diff
@@ -203,6 +203,22 @@ __attribute__((objc_subclassing_restricted)) @interface MDCActionSheetController
  */
 @property(nonatomic) UIImageRenderingMode imageRenderingMode;
 
+/**
+ Determines if a divider should be shown between the header and actions. To customize the divider
+ color see @c headerDividerColor.
+
+ Defaults to NO.
+ */
+@property(nonatomic, assign) BOOL showsHeaderDivider;
+
+/**
+ The color of the divider between the header and actions. This API should be used with
```

Comment:
Tip: You can probably remove the sentence, "This API should be used..." since I believe it's no longer required that updating this property is necessary to make the divider visible.

Reply:
Removed.